contracts/cw1/subkeys/useContract.ts

```diff
@@ -32,10 +32,8 @@ export function useCW1SubkeysContract(): UseCW1SubkeysContractProps {
   }, [])
 
   useEffect(() => {
-    if (wallet.initialized) {
-      const cw20BaseContract = initContract(wallet.getClient(), wallet.address)
-      setCW1Subkeys(cw20BaseContract)
-    }
+    const cw20BaseContract = initContract(wallet.getClient(), wallet.address)
+    setCW1Subkeys(cw20BaseContract)
   }, [wallet])
```

Comment:
Removing the if check works with cw1-subkeys execute page but for other contracts (cw20-base and cw721-base) it crashes the page since the if check is still there. Need to remove the code from those files.